code/modules/antagonists/cult/cult_items.dm

```diff
@@ -403,7 +403,7 @@ Striking a noncultist, however, will tear their flesh."}
 	armor = list(MELEE = 50, BULLET = 40, LASER = 50, ENERGY = 60, BOMB = 50, BIO = 100, FIRE = 100, ACID = 100)
 	hoodtype = /obj/item/clothing/head/hooded/cult_hoodie/hardened
 	clothing_flags = STOPSPRESSUREDAMAGE | THICKMATERIAL
-	flags_inv = HIDEGLOVES | HIDESHOES | HIDEJUMPSUIT
+	flags_inv = HIDEGLOVES | HIDESHOES | HIDEJUMPSUIT | HIDETAIL // SKYRAT EDIT ADDITION - HIDETAIL
```

Comment:
I really want override files for simple shit like this. I think an override module is needed.

Reply:
I can do that in a follow up, I think it's a little outside the scope of this PR particularly